sklearn/ensemble/tests/test_stacking.py

```diff
@@ -646,6 +654,120 @@ def fit(self, X, y):
         stacker.n_features_in_
 
 
+@pytest.mark.parametrize(
+    "estimator",
+    [
+        # output a 2D array of the probability of the positive class for each output
+        MLPClassifier(random_state=42),
+        # output a list of 2D array containing the probability of each class
+        # for each output
+        RandomForestClassifier(random_state=42),
+    ],
+    ids=["MLPClassifier", "RandomForestClassifier"],
+)
+def test_stacking_classifier_multilabel_predict_proba(estimator):
+    """Check the behaviour for the multilabel classification case and the
+    `predict_proba` stacking method.
+
+    Estimators are not consistent with the output arrays and we need to ensure that
+    we handle all cases.
+    """
+    X_train, X_test, y_train, y_test = train_test_split(
+        X_multilabel, y_multilabel, stratify=y_multilabel, random_state=42
+    )
+    n_outputs = 3
+
+    # MLPClassifier will return a 2D array where each column is the probability
+    # of the positive class for each output. We stack this array directly without
+    # any further processing.
```

Comment:
Actually we can remove this comment (it was before I parametrize the test). The info is already in the parametrization.